build-support/bin/generate_travis_yml.py

```diff
@@ -578,14 +577,13 @@ def rust_tests_osx() -> Dict:
     "name": "Rust tests - OSX",
     "os": "osx",
     # Fuse actually works on this image. It hangs on many others.
```

Comment:
Is this comment now outdated?

Reply:
Apparently it's not just outdated, it's wrong...  Dammit. 

How did we find that Fuse worked on 8.3? Trial and error? Travis make no claims about this.

Reply:
IIRC we originally were on 7.something and it didn't work, and 8.3 was the first we tried which worked by incrementing version numbers.